src/ape/api/projects.py

```diff
@@ -190,6 +190,13 @@ def version_id(self) -> str:
         Most often, this is either a version number or a branch name.
         """
 
+    @property
+    @abstractmethod
+    def url(self) -> AnyUrl:
```

Comment:
nit, but shouldn't this be `uri`?

Reply:
The type is `AnyUrl`, not `AnyUri`! JK, I am actually sure on the technically correct answer. Is there a difference asking for a friend?

I am happy to switch to `uri` though!